File: bench/bench.cpp
#include <iostream>

long long fib(long long n) {
    if (n < 2) return n;
    return fib(n - 1) + fib(n - 2);
}

int main() {
    std::cout << fib(32) << "\n";
    long long sum = 0;
    for (long long i = 0; i < 100000000; i++)
        sum += i % 7;
    std::cout << sum << "\n";
    return 0;
}

File: bench/bench.rs
fn fib(n: i64) -> i64 {
    if n < 2 { return n; }
    fib(n - 1) + fib(n - 2)
}

fn main() {
    println!("{}", fib(32));
    let mut sum: i64 = 0;
    for i in 0..100_000_000i64 {
        sum += i % 7;
    }
    println!("{}", sum);
}
